src/entrypoints/ui/utils/figma.ts

```diff
@@ -7,5 +7,8 @@ export function getCanvas() {
 
 export function getObjectsPanel() {
   // Similar to `getCanvas()`.
-  return document.querySelector('[data-testid="objects-panel"]') as HTMLElement
+  const node =
+    document.querySelector('[data-testid="objects-panel"]') ||
+    document.querySelector('[data-testid="devHandoffNodesList"]')
```

Comment:
Is this from Dev Mode? Are we expecting using the extension along with Dev Mode itself? I think most features are already included in Dev Mode.

Reply:
sorry, this is for development, because Figma remove the `window.figma`, if u mind, i can remove it

Reply:
OK let's remove it for now. You can now run the extension on a editable file in design mode to test.